packages/react/src/components/molecules/Tags/index.js

```diff
@@ -64,11 +65,13 @@ class Tags extends React.Component {
 Tags.propTypes = {
   /** The tags applied to the search list <br/>
         type: The type of tag <br />
-        text: The text displayed by the tag (required) <br />
+        text: The content displayed by the tag (required) <br />
+        description: The visually hidden description of the tag <br />
         value: The value of the tag  */
   tags: PropTypes.arrayOf(PropTypes.shape({
     type: PropTypes.string,
-    text: PropTypes.string.isRequired,
+    text: PropTypes.node.isRequired,
+    description: PropTypes.node,
```

Comment:
same comment here. Can we reference the ButtonTag.propTypes here instead? 
```
PropTypes.arrayOf(PropTypes.shape(ButtonWithIcon.propTypes))
```

Reply:
Unfortunately, we can't just reference `ButtonTag.propTypes` since not all of these props are allowed here. We control the `handleClick` prop in this component, so it can't be passed. Once Mayflower switches to TS, it will be possible to only take a a subset of the props. But, our JSCodeShift converter isn't smart enough to handle it at the moment, as I see.

Reply:
I see...ok